concreate/cli.py

```diff
@@ -61,7 +61,12 @@ def run(self):
                                  dest='build_osbs_release',
                                  action='store_true',
                                  default=False,
-                                 help='execute OSBS release build.')
+                                 help='execute OSBS release build')
+
+        parser.add_argument('--tech-preview',
```

Comment:
Shouldn't this be `--build-tech-preview`? It's a build flag.

Reply:
i'm not sure its basically generate option - but this should be hidden in future - so Im not sure here

Reply:
You even added it to the build documentation...